### DISCUSSION TITLE
Balloon feature from tkinter.tix not compactible

### DISCUSSION THREAD

**Discussion by saquibntt** [TomSchimansky/CustomTkinter#1528] [General]
Hello,

I wanted a balloon feature(i.e hovering text when placed cursor) in my GUI and seems its not supported with custom TKinter. 

Below is an example :-

```
#Import the tkinter library
from tkinter import *
from tkinter.tix import *
import customtkinter

#Create an instance of tkinter frame
win = Tk()
#win = customtkinter.CTk()  ############## Uncomment for testing ################
#Set the geometry
win.geometry("400x200")

#Create a tooltip
tip= Balloon(win)

#Create a Button widget
my_button=Button(win, text= "Python", font=('Helvetica bold', 20))
my_button.pack(pady=20)

#Bind the tooltip with button
tip.bind_widget(my_button,balloonmsg="Python is an interpreted, high-level")

win.mainloop()
```
ERROR:

> Traceback (most recent call last):
>   File "basic_form.py", line 229, in <module>
>     tip= Balloon(win)
>   File "/usr/lib/python3.8/tkinter/tix.py", line 525, in __init__
>     TixWidget.__init__(self, master, 'tixBalloon', static, cnf, kw)
>   File "/usr/lib/python3.8/tkinter/tix.py", line 311, in __init__
>     self.tk.call(widgetName, self._w, *extra)
> _tkinter.TclError: invalid command name "tixBalloon"

**Comment by Akascape** [comment]
@saquibntt
>hovering text when placed cursor

Try this: https://github.com/Akascape/CTkToolTip

**Reply by avalon60** [reply]
I've moved to CTkToolTip in my CTk Theme Builder app - it's excellent and has some nice features.
